app/models/manageiq/providers/google/network_manager/refresh_parser.rb

```diff
@@ -131,6 +219,107 @@ def get_network_ports
         process_collection(network_ports, :network_ports) { |n| parse_network_port(n) }
       end
 
+      def get_load_balancers
+        # GCE uses forwarding-rules rather than load-balancers
+        forwarding_rules = @connection.forwarding_rules.all
+
+        process_collection(forwarding_rules, :load_balancers) { |forwarding_rule| parse_load_balancer(forwarding_rule) }
+      end
+
+      def get_load_balancer_pools
+        # Right now we only support network-based load-balancers, instead of the
+        # more complicated HTTP/HTTPS load balancers.
+        # TODO(jsselman): Add support for http/https proxies
+        target_pools = @connection.target_pools.all
+
+        process_collection(target_pools, :load_balancer_pools) { |target_pool| parse_load_balancer_pool(target_pool) }
+        get_load_balancer_pool_members(target_pools)
+      end
+
+      def get_load_balancer_pool_members(target_pools)
+        @data[:load_balancer_pool_members] = []
+
+        target_pools.each do |tp|
+          lb_pool_members = tp.instances.collect { |m| parse_load_balancer_pool_member(m) }
+          lb_pool_members.each do |member|
+            @data_index.store_path(:load_balancer_pool_members, member[:ems_ref], member)
+            @data[:load_balancer_pool_members] << member
+          end
+          lb_pool = @data_index.fetch_path(:load_balancer_pools, tp.id)
+          lb_pool[:load_balancer_pool_member_pools] = lb_pool_members.collect do |member|
+            {:load_balancer_pool_member => member}
+          end
+        end
+      end
+
+      def get_load_balancer_listeners
+        # There's no explicit listener concept in GCE, so again we reuse the
+        # forwarding rule.
+        forwarding_rules = @connection.forwarding_rules.all
+
+        process_collection(forwarding_rules, :load_balancer_listeners) do |forwarding_rule|
+          parse_load_balancer_listener(forwarding_rule)
+        end
+      end
+
+      def parse_load_balancer(forwarding_rule)
+        uid = forwarding_rule.id
+
+        new_result = {
+          :type    => ManageIQ::Providers::Google::NetworkManager::LoadBalancer.name,
+          :ems_ref => uid,
+          :name    => forwarding_rule.name
+        }
+
+        return uid, new_result
+      end
+
+      def parse_load_balancer_pool(target_pool)
+        uid = target_pool.id
+
+        new_result = {
+          :type    => ManageIQ::Providers::Google::NetworkManager::LoadBalancerPool.name,
+          :ems_ref => uid,
+          :name    => target_pool.name
+        }
+
+        @target_pool_index[target_pool.self_link] = new_result
+
+        return uid, new_result
+      end
+
+      def parse_load_balancer_pool_member(member_link)
+        vm_id = get_vm_id_from_link(member_link)
+        {
+          :type    => "ManageIQ::Providers::Google::NetworkManager::LoadBalancerPoolMember",
+          :ems_ref => Digest::MD5.base64digest(member_link),
```

Comment:
@Ladas here is a good example where we need a unique value but there isn't really a native `:ems_ref` type